src/content/docs/merge-queue/parallel-checks.mdx

```diff
@@ -133,6 +133,34 @@ Parallel Checks adapt automatically as your code and rules evolve:
 
 ## Important Considerations
 
+### In‑place checks (no drafts)
+
+Mergify runs checks directly on the original pull request (instead of using
+temporary draft PRs) only when ALL of the following are true:
+
+- `queue_rules[*].batch_size = 1`
+- `merge_queue.max_parallel_checks = 1`
+- No two‑step CI is configured (i.e., no separate `merge_conditions` beyond
+  `queue_conditions` in your queue rules)
+
+There is no separate `allow_inplace_checks` setting. In‑place checks are
```

Comment:
Maybe don't talk about things that do not exist? :)

Reply:
<img width="266" height="189" alt="Image" src="https://github.com/user-attachments/assets/e5473820-7cdc-4d57-ac50-c344ff2bc6fc" />